src/stamp/preprocessing/extractor/gigapath.py

```diff
@@ -0,0 +1,31 @@
+try:
+    import timm
+    from torchvision import transforms
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "gigapath dependencies not installed."
+        " Please reinstall stamp using `pip install 'stamp[gigapath]'`"
+    ) from e
```

Comment:
Please add these dependencies as a dependency group in the toml file: https://github.com/EzicStar/STAMP/blob/0b4760cdecd71e04cad5667d450ccbab70bbbec2/pyproject.toml#L47-L72